btk/create_observing_generator.py

```diff
@@ -9,11 +43,14 @@ def default_obs_conditions(Args, band):
         Args: Class containing parameters to generate blends
```

Comment:
I think Ismael has already opened an issue to remove the Args in order to get a cleaner code, so it probably will not stay there that long. I will add something to the docstring for the time being.

Reply:
Great! If we already fix the places that need this it will make the job easier later down the line. Here, it's just giving the precise name of the class for clarity. Thanks.